src/main/java/carpet/script/api/Inventories.java

```diff
@@ -424,7 +424,7 @@ else if (owner instanceof LivingEntity villager)
                 return player;
             });
             if(lv.get(1) instanceof ScreenHandlerValue screenHandlerValue) {
-                if(!screenHandlerValue.hasInventory() && playerValueList.size() != 1) {
+                if(playerValueList.size() != 1) {
                     throw new InternalExpressionException("screen handler type '" + screenHandlerValue.getString() + "' does not support multiple players.");
```

Comment:
Suggested error message: "Screen handles do not support multiple players"

Reply:
yeah most of that code can be removed anyway, since its no longer expecting a list of players

Reply:
in fact, the open_screen function and create_screen can be combined, since there is no longer a reason to not immediately display a screen